modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/A11yTool.testcase

```diff
@@ -161,13 +153,15 @@ definition {
 				key_text = "critical",
 				locator1 = "A11yTool#HIGHLIGHTED_ELEMENT_POPOVER_SEVERITY");
 
+			AssertElementPresent(locator1 = "A11yTool#HIGHLIGHTED_ELEMENT_ICON");
+		}
+
+		task ("Assert minor, moderate, serious violations are not highlighted") {
 			for (var severity : list "minor,moderate,serious") {
```

Comment:
I'm having a hard time with seeing this for loop on the testcase level. I'm not sure if we need to loop it at all and would be more clear/readable if we just list out AssertElementNotPresent for each severity. Could you make this last change for this and the other for loops? It will be extra clear and readable if we directly show how many assertions we're making in the testcase.

Reply:
you got it captain :boat: